### ISSUE TITLE
Can't rightclick (use) items in my bags

### ISSUE THREAD

**Issue by kpontussvensson** [Andols0/BagSearchPlus#2]
Hey!

I get the "BagSearchPlus has been blocked from an action only available to the Blizzard UI." popup when I try to use Hearthstone/flasks/etc in my bags after I've been in combat. I can use items after a reload if I haven't been in combat before, but as soon as I've been in combat once the problem occurs with the following lua error:

```
3x [ADDON_ACTION_FORBIDDEN] AddOn 'BagSearchPlus' tried to call the protected function 'UNKNOWN()'.
[string "@Interface/AddOns/!BugGrabber/BugGrabber.lua"]:480: in function <Interface/AddOns/!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: ?
[string "=[C]"]: in function `UseContainerItem'
[string "@Interface/FrameXML/ContainerFrame.lua"]:1384: in function `ContainerFrameItemButton_OnClick'
[string "@Interface/FrameXML/ContainerFrame.lua"]:1435: in function <Interface/FrameXML/ContainerFrame.lua:1423>
```

When I turn BagSearchPlus off it works with no issues. I'm using addon version 10.0.0.

**Comment by Andols0** [comment]
Hello!

Thats not good. I'll look into it this week. 

Are you also using any other bag addon or ElvUi?

**Comment by kpontussvensson** [comment]
Nope, I'm using default UI. I've got BagSync as well but I get the error with BagSync disabled. I can check later on if I get the error with only BagSearchPlus enabled.

**Comment by kpontussvensson** [comment]
Update:

I've tracked it down to not actually having to do with dropping combat or not. I can 100% replicate this with only BagSearchPlus enabled by doing this:

1. Open my bags
2. Close my bags
3. Open my bags
4. Click an item

**Comment by Andols0** [comment]
Could you try to open Bsp.lua and add
-- to line 97 https://github.com/Andols0/BagSearchPlus/blob/59e0659d0ff0f0cf2ffc7106dfa8eb3b0b80265a/Bsp.lua#L97

That might help. Currently only on my phone out of country 😅

**Comment by kpontussvensson** [comment]
~~Yep, that fixed it for me!~~

Nvm! I got the same error now, just waited a bit and tried to use Hearthstone again and got it.

EDIT: I reloaded again (I reloaded after editing the file as well, so I had the change active before) and now I haven't gotten it yet. Will keep you updated.

EDIT 2: I still can't replicate the issue, it seems to have fixed it even though I reloaded the first time as well. 😄
